profile_library/jbl/Link Music/model.json

```
@@ -0,0 +1,31 @@
+{
+  "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+  "calculation_strategy": "linear",
+  "created_at": "2025-02-10T16:32:20.219347",
+  "device_type": "smart_speaker",
+  "linear_config": {
+    "calibrate": [
+      "0 -> 2.71",
+      "10 -> 2.72",
+      "20 -> 2.76",
+      "30 -> 2.76",
+      "40 -> 2.76",
+      "50 -> 2.76",
+      "60 -> 2.83",
+      "70 -> 3.47",
+      "80 -> 5.49",
+      "90 -> 7.44",
+      "100 -> 8.56"
+    ]
+  },
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Nous A8T (esphome)",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.6:docker"
+  },
+  "name": "JBL Link Music",
+  "standby_power": 1.7
```

```suggestion
  "standby_power": 1.7,
  "aliases": ["JBL Link Music"]
```

`JBL Link Music` must be added as an alias here, as that is the full model as returned from HA device info.